src/translations/lang/en.json

```diff
@@ -976,6 +976,15 @@
     "NO_NON_VIEW_ONLY_ACCOUNTS": "You currently only have $link_support addresses added to MyCrypto, which you cannot send from. Please $link_add_account.",
     "HERE": "here",
     "ADD_AN_ACCOUNT": "add an account",
-    "VIEW_ONLY_ADDRESSES": "View-Only addresses"
+    "VIEW_ONLY_ADDRESSES": "View-Only addresses",
+    "TX_TYPE": "Transaction Type",
+    "MAX_FEE_PER_GAS": "Max Fee",
+    "MAX_PRIORITY_FEE": "Max Priority Fee",
+    "FEE_RANGE_ESTIMATE": "Fee Range Estimate",
+    "RECOMMENDED_TOTAL_FEE": "Recommended Total Fee",
+    "CURRENT_BASE_FEE": "Current Base Fee: $fee",
+    "SAVE": "Save",
+    "PRIORITY_FEE_MAX_ERROR": "Priority Fee must be smaller than or equal to Max Fee",
+    "GWEI": "GWEI"
```

Comment:
"Gwei"

Reply:
This is intentional. Would you prefer to have it lowercase and use `toUpperCase` when used?

Reply:
I mean that it's commonly written as Gwei rather than GWEI, so I think we should change that.